librz/core/cconfig.c

```diff
@@ -2574,10 +2574,17 @@ static bool cb_analysis_roregs(RzCore *core, RzConfigNode *node) {
 	return true;
 }
 
+static bool cc_print(void *p, const char *k, const char *v) {
+	if (!strcmp(v, "cc")) {
+		rz_cons_println(k);
+	}
+	return true;
+}
+
 static bool cb_analysissyscc(RzCore *core, RzConfigNode *node) {
 	if (core && core->analysis) {
 		if (!strcmp(node->value, "?")) {
-			rz_core_cmd0(core, "afcl");
+			sdb_foreach(core->analysis->sdb_cc, cc_print, NULL);
```

Comment:
I would extract `afcl` in separate function instead of copying `cc_print` around. That would be useful also when we are going to move these commands to newshell. Keep that in mind while refactoring, because every command we extract in a simple API it's one more command that becomes very easy to convert.

Reply:
Done. Created a `RZ_IPI rz_core_analysis_calling_conventions_print(RzCore *core)` function.